rllib/agents/ppo/ppo_torch_policy.py

```diff
@@ -255,12 +255,11 @@ def setup_mixins(policy: Policy, obs_space: gym.spaces.Space,
     LearningRateSchedule.__init__(policy, config["lr"], config["lr_schedule"])
 
 
-def training_view_requirements_fn(
-        policy: Policy) -> Dict[str, ViewRequirement]:
-    """Function defining the view requirements for training the policy.
+def view_requirements_fn(policy: Policy) -> Dict[str, ViewRequirement]:
```

Comment:
Renaming this back to just `view_requirements`, b/c it also includes requirements for post-processing (not just training).